crates/astria-sequencer/src/grpc/sequencer.rs

```diff
@@ -117,15 +118,22 @@ impl SequencerService for SequencerServer {
                 ))
             })?;
 
-        let (rollup_transactions_proof, rollup_ids_proof) = snapshot
-            .get_block_proofs_by_block_hash(&block_hash)
+        let rollup_transactions_proof = snapshot
+            .get_rollup_transactions_proof_by_block_hash(&block_hash)
             .await
             .map_err(|e| {
                 Status::internal(format!(
-                    "failed to get sequencer block proofs from storage: {e}"
+                    "failed to get rollup transactions proof from storage: {e}"
                 ))
             })?;
 
+        let rollup_ids_proof = snapshot
+            .get_rollup_ids_proof_by_block_hash(&block_hash)
+            .await
+            .map_err(|e| {
+                Status::internal(format!("failed to get rollup ids proof from storage: {e}"))
```

Comment:
This will likely underreport the error. Ideally we log this, but let's change this to `{e:#}` for the time being?

Reply:
I think there's no difference in these two forms now we've replaced `anyhow` with our own `eyre` wrapper.

Reply:
Yeah, you are right.